ethcore/src/evm/interpreter/mod.rs

```diff
@@ -126,7 +126,7 @@ impl<Cost: CostType> evm::Evm for Interpreter<Cost> {
 			gasometer.current_gas = gasometer.current_gas - gas_cost;
 
 			evm_debug!({
-				println!("[0x{:x}][{}(0x{:x}) Gas: {:x}\n  Gas Before: {:x}",
+				println!("[0x{:x}][{}(0x{:x}) Gas: {:?}\n  Gas Before: {:?}",
```

Comment:
It's much easier to run tests when there are printlns, cause logs are not initialized in tests. It's also a compilation feature, so when someone is compiling with it I guess you would expect the logs to be always printed.